website/docs/user_attributes.md

```diff
@@ -0,0 +1,166 @@
+|         **description** | The user's legal address                                                                        |
+|   **SAML Attribute(s)** | `http://eidas.europa.eu/attributes/legalperson/LegalAddress` (LegalAddress)                     |
+|          **OIDC scope** | `legal_address`                                                                                 |
+|       **OIDC claim(s)** | `legal_address`                                                                                 |
+| **OIDC claim location** | Userinfo endpoint                                                                               |
+|              **origin** | Provided by user's Identity Provider                                                            |
+|             **changes** | Yes                                                                                             |
+|        **multiplicity** | Single-valued                                                                                   |
+|        **availability** | Always                                                                                          |
+|             **example** | _PGVpZGFzLWxlZ2FsOkZ1bGxDdmFkZHJlc3M+TGVnYWwgYWRkcmVzczwvZWlkYXMtbGVnYWw6RnVsbEN2YWRkcmVzcz4K_  |
+|               **notes** | -                                                                                               |
+|              **status** | Stable                                                                                          |
+
+### 10. Vat Registration
+
+|          attribute name | Vat Registration                                                                               |
+| ----------------------: | :--------------------------------------------------------------------------------------------- |
+|         **description** | The user's vat registration number                                                             |
+|   **SAML Attribute(s)** | `http://eidas.europa.eu/attributes/legalperson/VATRegistrationNumber` (VATRegistrationNumber)  |
+|          **OIDC scope** | `vat_registration`                                                                             |
+|       **OIDC claim(s)** | `vat_registration`                                                                             |
+| **OIDC claim location** | Userinfo endpoint                                                                              |
+|              **origin** | Provided by user's Identity Provider                                                           |
+|             **changes** | Yes                                                                                            |
+|        **multiplicity** | Single-valued                                                                                  |
+|        **availability** | Always                                                                                         |
```

Comment:
availability: Only when released by the eIDAS Node from which the legal person is authenticating